pandas/tests/indexes/multi/test_indexing.py

```diff
@@ -397,3 +397,15 @@ def test_get_indexer_categorical_time():
          Categorical(date_range("2012-01-01", periods=3, freq='H'))])
     result = midx.get_indexer(midx)
     tm.assert_numpy_array_equal(result, np.arange(9, dtype=np.intp))
+
+
+def test_timestamp_multiindex_indexer():
+    # https://github.com/pandas-dev/pandas/issues/26944
+    idx = pd.MultiIndex.from_product([
+        pd.date_range("2019-01-01T00:15:33", periods=100, freq="H",
+                      name="date"),
+        ['x'],
+        [3]
+    ])
+    df = pd.DataFrame({'foo': np.arange(len(idx))}, idx)
+    assert df.loc[pd.IndexSlice['2019-1-2':, "x", :], 'foo'][-1] == 98
```

Comment:
Could you use `tm.assert_series_equal` to test the resulting `Series` produced by the slice instead of the last value?

Reply:
yes, I've changed that.